texturepacker/editor/src/texturepacker.clj

```diff
@@ -1301,7 +1301,7 @@
             (g/operation-sequence op-seq)
             (app-view/select app-view image-nodes)))))))
 
-(handler/defhandler :add-from-file :workbench
+(handler/defhandler :edit.add-referenced-component :workbench
```

Comment:
Hmm, I can see why these commands were named a bit vague seeing as how they do not operate on components in a few places.